src/ViewerComponents/ImageViewer.svelte

```diff
@@ -0,0 +1,120 @@
+<script>
+    import {onMount} from 'svelte';
+    import {token} from "../store";
+    import OpenSeadragon from 'openseadragon';
+
+    export let resource, ontology, server, user, shortname, shortcode;
+    let error = false;
+    let container
+    let viewer
+
+    onMount(() => {
+        container.id = Math.random().toString(36).substring(8);
```

Comment:
What happens if two ViewerComponents on one page randomly get assigned the same id?

Reply:
Thats a valid point. I make the id longer so the chance will be decreased of a collision